Gemfile.lock

```diff
@@ -417,20 +417,20 @@ GEM
       coercible (~> 1.0)
       descendants_tracker (~> 0.0, >= 0.0.3)
       equalizer (~> 0.0, >= 0.0.9)
-    warden (1.2.8)
-      rack (>= 2.0.6)
+    warden (1.2.9)
```

Comment:
### [warden](https://github.com/hassox/warden)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.8 → 1.2.9

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/wardencommunity/warden/compare/v1.2.8...v1.2.9).

These are the first 10 commits:
- (e0c3b41) [change warden wiki link in README.md](https://github.com/wardencommunity/warden/commit/e0c3b413acb658c5b40dc2ab04c0e23ee3463d04)
- (ef0b1f4) [Merge pull request #172 from buncismamen/master](https://github.com/wardencommunity/warden/commit/ef0b1f4f44d7cc0e738b7e40c049d89e60351a51)
- (ed160f6) [Fix document comment[ci skip]](https://github.com/wardencommunity/warden/commit/ed160f6738d8a00333c9517e3f5170321a13566a)
- (902903a) [Add newer ruby versions for CI](https://github.com/wardencommunity/warden/commit/902903a12bd261809ce8e59fa59e8e031765349c)
- (fdb7189) [Merge pull request #174 from colorbox/add_ruby_version_on_CI](https://github.com/wardencommunity/warden/commit/fdb71899c8c85f25e2259e957edbdd15dfe36024)
- (df575ff) [Merge branch 'master' into fix_document](https://github.com/wardencommunity/warden/commit/df575ff21c7a3598043e0a67daafb727482deb80)
- (b7ff0f4) [Merge pull request #173 from colorbox/fix_document](https://github.com/wardencommunity/warden/commit/b7ff0f4ffacd7ad5a9b5a0191f1da0c7a64b2c2c)
- (c1af24f) [Set `winning_strategy` when `:warden` is thrown](https://github.com/wardencommunity/warden/commit/c1af24f556b5faf4f7f1acfe49a02f56b3c5c567)
- (7aec137) [Merge pull request #175 from landongrindheim/set-winning-str…](https://github.com/wardencommunity/warden/commit/7aec1374cba3409d82208c5c7fb0c98f18a0c2c1)
- (3110169) [gemspec: rubyforge_project is removed](https://github.com/wardencommunity/warden/commit/31101697acc19aa7f6bf3f07f6d462eb7446fa63)


</details>